src/libmr_commands.c

```diff
@@ -15,10 +15,22 @@
 static inline bool check_and_reply_on_error(ExecutionCtx *eCtx, RedisModuleCtx *rctx) {
     size_t len = MR_ExecutionCtxGetErrorsLen(eCtx);
     if (unlikely(len > 0)) {
-        RedisModule_ReplyWithError(rctx, "multi shard cmd failed");
         RedisModule_Log(rctx, "warning", "got libmr error:");
+        bool max_idle_reached = false;
         for (size_t i = 0; i < len; ++i) {
             RedisModule_Log(rctx, "warning", "%s", MR_ExecutionCtxGetError(eCtx, i));
+            if (!strcmp("execution max idle reached", MR_ExecutionCtxGetError(eCtx, i))) {
+                max_idle_reached = true;
```

Comment:
can add a break?

Reply:
@oshadmi I still would like to log all errors from libmr